src/Data/TypeRepMap/Internal.hs

```diff
@@ -390,14 +387,19 @@ fromTriples kvs = TypeRepMap (GHC.fromList fpAs) (GHC.fromList fpBs) (GHC.fromLi
 ----------------------------------------------------------------------------
 
 fromSortedList :: forall a . [a] -> [a]
-fromSortedList l = IM.elems $ fst $ go 0 0 mempty (IM.fromList $ zip [0..] l)
+fromSortedList l = runST $ do
+    let n = length l
+    let arrOrigin = fromList l
+    arrResult <- thawArray arrOrigin 0 n
+    _ <- go n 0 0 arrResult arrOrigin
```

Comment:
`() <$` 😏

Reply:
it's called `void` (from `Control.Monad`)